### DISCUSSION TITLE
May I ask which papers S2STransformerBeamSearch refers to?

### DISCUSSION THREAD

**Discussion by My-captain** [speechbrain/speechbrain#1907] [General]
Papers abount **_S2STransformerBeamSearch_**？

**Comment by Adel-Moumen** [comment]
Hi, S2STransformerBeamSearch is based on the Hybrid CTC/Att joint decoding (http://zhaoshuaijiang.com/file/Hybrid_CTC_Attention_Architecture_for_End-to-End_Speech_Recognition.pdf) beam search.

**Reply by My-captain** [reply]
Thanks!😊
